examples/spark/zeppelin-controller.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
         - name: zeppelin
-          image: gcr.io/google_containers/zeppelin:v0.5.5_v1
+          image: gcr.io/google_containers/zeppelin:v0.5.5_v2
```

Comment:
Is this tag the standard way we're going to version images?  Dunno if there is convention here.